doc/rados/configuration/ceph-conf.rst

```diff
@@ -622,7 +622,7 @@ will report the value of a single option.
 Changes since Nautilus
 ======================
 
-With the Octopus release We changed the way the configuration file is parsed.
+With the Octopus release ee changed the way the configuration file is parsed.
```

Comment:
?

Reply:
@epuertat The `We` shouldn't have been capitalized, nice catch, but it looks like you inadvertently replaced the `W` with `e` instead of `w`.

Don't you hate it when a PR changing a single letter has a mistake in it?  Happened to me last month 🤐

Reply:
Fixed! Thanks a lot @batrick and @anthonyeleven !